opty/direct_collocation.py

```diff
@@ -459,13 +462,18 @@ def plot_constraint_violations(self, vector, axes=None):
         - r : number of unknown parameters
         - s : number of unknown time intervals
 
+        If node_time_interval is a sympy.Symbol, the times in the instance
+        constraints should be given as alpha * h, where alpha is an integer
+        or a float and h is the name of the node_time_interval symbol. This
+        will ensure that the time is given correctly in the plot.
```

Comment:
This doesn't belong in the docstring of `plot_constraint_violations`. If you think more explanation is needed on how to work with a variable time interval, that should be documented in the `ConstraintCollocator` primary docstring.

Reply:
> This doesn't belong in the docstring of `plot_constraint_violations`. If you think more explanation is needed on how to work with a variable time interval, that should be documented in the `ConstraintCollocator` primary docstring.

I can move it there, or delete it. What is your preference? After all, YOUR program!

Reply:
I welcome improvements to the documentation and you think it needs better or more explanation you can add it. But maybe just do it in another PR as it is unrelated to this change.

Reply:
> I welcome improvements to the documentation and if you think it needs better or more explanation you can add it. But maybe just do it in another PR as it is unrelated to this change.

I will remove it. What I did is a minuscule thing in opty, and explanation likely not needed.